Domains/Civil/CivilPhysical/CivilPhysical.remarks.md

```diff
@@ -7,17 +7,15 @@ remarksTarget: CivilPhysical.ecschema.md
 
 This schema contains classes that are commonly used in various types of Civil projects, including Road, Rail and Site-related.
 
-NOTE: Currently under development. This schema should not be used for production workflows. Data created using this schema is not supported and may not be upgradable.
+![CivilPhysical](./media/CivilPhysical-classes.png)
 
 ## Entity Classes
 
-![CivilPhysical](./media/CivilPhysical-classes.png)
-
 ### CourseType
 
 Instances of `CourseType` provide an additional classification that can be applied to `Course`s. Examples include Pavement, Ballast and Protection. An instance of `CourseType` can optionally specify a single *Physical Material* via its `PhysicalMaterial` property.
 
-Equivalent to [IfcCourseType](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC2/HTML/link/ifccoursetype.htm).
+Equivalent to [IfcCourseType](http://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcCourseType.htm).

```

Comment:
can this like and the others below be https?

Reply:
No, the https versions of them don't work. The previous ones were about a release candidate, with docs under https - not sure why buildingSMART did it that way.